packages/web/components/swap-tool/index.tsx

```diff
@@ -812,119 +953,128 @@ export const SwapTool: FunctionComponent<{
 
             <SkeletonLoader
```

Comment:
I had missed this comment, the loading animation part we didn't touch, or maybe the new header causes some misalignment? What am I missing?

Reply:
I'm talking about the expandable details section below the inputs. When loading it appears differently to what we currently have in stage.

Reply:
![Screenshot 2023-10-11 at 10 11 24 AM](https://github.com/osmosis-labs/osmosis-frontend/assets/4606373/06fc9e59-c102-4bc9-a189-6fd5264acc28)

Reply:
We did this because in the new design there are changes to the component the bottom section when collapsed is large compared to before and shows more information

<img width="379" alt="Screenshot 2023-10-11 alle 16 37 19" src="https://github.com/osmosis-labs/osmosis-frontend/assets/17269969/028eb775-3c1c-4f54-9252-f193f4c38788">

Reply:
Can we leave that area as it is? I'll sync with Syed but I think that was unintentional, and changes there are not worth it IMO

Reply:
Yes, I'll reverse it

Reply:
Done here: https://github.com/osmosis-labs/osmosis-frontend/pull/2234/commits/8e793e257f6db62a2e57d9ab21747594af5fe721